builtin/providers/azure/resource_azure_sql_database_service.go

```diff
@@ -0,0 +1,235 @@
+package azure
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/management/sql"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+// resourceAzureSqlDatabaseService returns the *schema.Resource
+// associated to an SQL Database Service on Azure.
+func resourceAzureSqlDatabaseService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAzureSqlDatabaseServiceCreate,
+		Read:   resourceAzureSqlDatabaseServiceRead,
+		Update: resourceAzureSqlDatabaseServiceUpdate,
+		Exists: resourceAzureSqlDatabaseServiceExists,
+		Delete: resourceAzureSqlDatabaseServiceDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"collation": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Good catch; also noticed a couple of other discrepancies in the argument definitions and fixed those as well.